fastestimator/op/tensorop/augmentation/mixup_batch.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Any, Dict, List, TypeVar, Union
+
+import tensorflow as tf
+import tensorflow_probability as tfp
+import torch
+
+from fastestimator.op.tensorop import TensorOp
+
+Tensor = TypeVar('Tensor', tf.Tensor, torch.Tensor)
+
+
+class MixUpBatch(TensorOp):
+    """ This class should be used in conjunction with MixUpLoss to perform mix-up training, which helps to reduce
```

Comment:
new doctoring format has short summary (less than 1 line) as first line, then more explanation in subsequent paragraph